src/iperf_api.c

```diff
@@ -799,6 +836,7 @@ iperf_parse_arguments(struct iperf_test *test, int argc, char **argv)
         {"udp", no_argument, NULL, 'u'},
         {"bitrate", required_argument, NULL, 'b'},
         {"bandwidth", required_argument, NULL, 'b'},
+	{"btotal", required_argument, NULL, OPT_TOTAL_BITRATE},
```

Comment:
For this new option flag, what about `server-bitrate-limit` or `server-total-bitrate`?  `btotal` is a bit terse.